src/backend/vulkan/src/command.rs

```diff
@@ -224,8 +241,8 @@ impl com::RawCommandBuffer<Backend> for CommandBuffer {
                 self.raw, // commandBuffer
                 conv::map_pipeline_stage(stages.start),
                 conv::map_pipeline_stage(stages.end),
-                vk::DependencyFlags::empty(), // dependencyFlags // TODO
-                &[],
+                mem::transmute(dependencies),
```

Comment:
Could we use `dependencies.bits()` or does it not exist here?

Reply:
It expects a bit-like opaque [structure](https://docs.rs/ash/0.21.1/ash/vk/types/struct.DependencyFlags.html) here that doesn't have `from_bits` (yet?). I'd honestly prefer ash just use the stock `bitflags` crate.

Reply:
Oh ok, makes sense